docs/source/guides/contribution_guide.rst

```diff
@@ -80,6 +80,36 @@ Optionally, to run the full, slow, verbose test suite run:
 
   (nucypher)$ pytest
 
+Setup Commit & Push Hooks
+--------------------------
+
+Pre-commit and pre-push are used for quality control by preventing a commit that will fail
+if passed along to CI servers or by making small formatting changes directly to source code files.
```

Comment:
```suggestion
`Pre-commit <https://pre-commit.com/>`_ and pre-push are used for quality control to identify and prevent the inclusion of problematic code changes. It may prevent a commit that will fail
if passed along to CI servers or make small formatting changes directly to source code files.
```

Reply:
^ Updated "It" to "They"